### PR TITLE
remove NEC

### PR DESCRIPTION
**Pull request by Cartrigger**
May cause mods to crash or break (not good mod)

### CODE REVIEW

**Comment by Flopgop** [comment]
How?

**Comment by Cartrigger** [comment]
some mods straight up fail when NEC is loaded

**Comment by Cartrigger** [comment]
this could be a user added mod, not core

**Comment by Flopgop** [comment]
> some mods straight up fail when NEC is loaded

any kind of log?

**Comment by Cartrigger** [comment]
iris is one of them
i need to find my old modpack so then i can see what mod is crashing due to NEC

**Comment by Cartrigger** [comment]
just look at iris's startup when NEC is loaded, tells you a bit

File: mods.json [hunk 1]
```diff
@@ -187,11 +182,7 @@
 			"slug": "immediatelyfast",
 			"version": "1.1.6",
 			"download_link": "https://cdn.modrinth.com/data/5ZwdcRci/versions/zImGOGqD/ImmediatelyFast-1.1.6%2B1.18.2.jar"
-		},
-		{
-			"slug": "notenoughcrashes",
-			"version": "4.2.0",
-			"download_link": "https://cdn.modrinth.com/data/yM94ont6/versions/lWmXt6Pj/notenoughcrashes-4.2.0%2B1.18.2-fabric.jar"
+		}
 		},
```
Comment (Flopgop) [inline mods.json:186]: Invalid Syntax @Cartrigger please fix.

**Review by Flopgop — CHANGES_REQUESTED** [review]
Invalid Syntax in json file.

**Comment by Cartrigger** [comment]
fixed it

**Pull request merged**